common/src/main/java/com/wynntils/utils/StringUtils.java

```diff
@@ -19,6 +19,7 @@
 
 public final class StringUtils {
     private static final String[] SUFFIXES = {"", "k", "m", "b", "t"}; // kilo, million, billion, trillion (short scale)
+    private static final long[] SUFFIX_MULTIPLIERS = {-1L, 1_000L, 1_000_000L, 1_000_000_000L, 1_000_000_000_000L};
```

Comment:
Why not 1L? I see that it is not used, but multiplying with 1L wouldn't mess up any math, even if you miss-handled something.